v0/destinations/hs/util.js

```diff
@@ -17,7 +133,168 @@ const getEmailAndUpdatedProps = properties => {
   };
 };
 
+/* NEW API util functions */
+
+/**
+ * look for the contact in hubspot and extract its contactId for updation
+ * @param {*} destination
+ * @returns
+ */
+const searchContacts = async (message, destination, lookupField = null) => {
+  const { Config } = destination;
+  let res;
+  let contactId = null;
+  const traits = getFieldValueFromMessage(message, "traits");
+  let propertyName;
+
+  if (lookupField) {
+    propertyName = lookupField;
```

Comment:
when lookup field can come as `email` then lets keep room for `getFieldValueFromMessage`